mhs/Pipfile

```diff
@@ -10,19 +10,11 @@ unittest-xml-reporting = "~=2.5"
 [packages]
 requests = "*"
 integration-adaptors-common = {editable = true,path = "./../common"}
-ldap3 = "*"
 tornado = "*"
-aioboto3 = "*"
 defusedxml = "~=0.6"
```

Comment:
This Pipfile is used just for the integration tests now yes, There was an issue with imports as well after merging in develop so I've moved the pipfile to the `selenium_tests` module as well since nothing else used it. 

As suggested I updated the packages to be dev packages as well